internal/cmd/rename.go

```diff
@@ -47,17 +47,17 @@ When run on a perennial branch:
 - confirm with the "--force"/"-f" option
 - registers the new perennial branch name in the local Git Town configuration`
 
-func renameBranchCommand() *cobra.Command {
+func renameCommand() *cobra.Command {
 	addDryRunFlag, readDryRunFlag := flags.DryRun()
 	addForceFlag, readForceFlag := flags.Force("force rename of perennial branch")
 	addVerboseFlag, readVerboseFlag := flags.Verbose()
 	cmd := cobra.Command{
-		Use:   "rename-branch [<old_branch_name>] <new_branch_name>",
+		Use:   "rename [<old_branch_name>] <new_branch_name>",
```

Comment:
@kevgo 2 questions:

1. Should we keep an undocumented `rename-branch` command so this isn't a breaking change?
2. Does Git Town have the ability to update the local Git aliases from `git rename-branch` to `git rename` (or detect old alises and suggest that the user update them)?

Reply:
Re 1: yes, I'll add that separately
Re 2: That's a great idea, will also implement separately

Reply:
Agree with the second point. I was thinking that day how git-town would handle changes in aliases.

Reply:
Git Town already automatically updates the configuration settings stored in Git metadata when their names change. We can use this mechanism to update the aliases, which are also stored in Git metadata.